openmmapi/include/openmm/internal/vectorize_sse.h

```diff
@@ -32,7 +32,7 @@
  * USE OR OTHER DEALINGS IN THE SOFTWARE.                                     *
  * -------------------------------------------------------------------------- */
 
-#include <smmintrin.h>
+#include <immintrin.h>

```

Comment:
Are you sure this will work when AVX is disabled?

Reply:
Should work as I never use other than this, although I don't use old compilers so maybe they break.

vectorize_sse is used by vectorize8, so when used in that context I wanted to be able to call AVX intrinsics. Using just the original smmintrin file it would have to change to:

#ifdef __AVX__
#include <immintrin.h>
#else 
#include <smmintrin.h>
#endif

Happy to do that if you'd prefer.

Reply:
That's probably safest, thanks.